src/device.cpp

```diff
@@ -125,6 +126,10 @@ void cvk_device::init_features(VkInstance instance) {
         VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_FLOAT16_INT8_FEATURES_KHR;
     m_features_variable_pointer.sType =
         VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_VARIABLE_POINTERS_FEATURES;
+    m_features_8bit_storage.sType =
+        VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_8BIT_STORAGE_FEATURES;
```

Comment:
Should this be VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_8BIT_STORAGE_FEATURES_KHR (KHR suffix) for older implementations? Same for 16-bit.

Reply:
Yes, will fix.

Reply:
Done, also did the variable pointers one for consistency.